lib/jekyll-github-metadata/repository.rb

```diff
@@ -98,6 +98,10 @@ def releases
         memoize_value :@releases, Value.new(proc { |c| c.releases(nwo) })
       end
 
+      def commits
+        memoize_value :@commits, Value.new(proc { |c| c.commits(nwo) })
```

Comment:
My only concern here is that we limit this to something reasonable like 50 or 100. I believe we may have `auto_paginate` on which would mean if you have 200,000 commits, we'd try to grab all of them. 😳